README.md

```diff
@@ -32,22 +44,26 @@ It is defined using a [docker-compose file](../docker-compose.yml) for developme
 
 ### The storage & transcoding environment
 
-Source files (video, subtitles,...) are directly uploaded to an `S3` bucket by instructors, which triggers `MediaConvert` to generate all necessary video files (various formats and fragments & manifests for adaptive-bitrate streaming) into a destination `S3` bucket. Those files are then served through the `CloudFront` CDN.
+Source files (video, documents, subtitles,...) are directly uploaded to an `S3` bucket by instructors. Depending the uploaded resource a lambda will be triggered to do diffenrent jobs:
+- Launch `MediaConvert` to generate all necessary video files (various formats and fragments & manifests for adaptive-bitrate streaming) into a destination `S3` bucket. Those files are then served through the `CloudFront` CDN.
```

Comment:
```suggestion
1. Launch AWS `MediaConvert` to generate all necessary video files (various formats and fragments & manifests for adaptive-bitrate streaming) into a destination AWS `S3` bucket. Those files are then served through the AWS `CloudFront` CDN.
```

Reply:
This is not an ordered list

Reply:
Ok, then do as you please. It's about enumerating choices, not ordering things.